site/source/tutorials/9001_dojo1_dijits/index.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: tutorial
+title: Using Dojo 1 Dijits
+overview: In this tutorial, you will learn how to use Dojo 1 Dijits in a Dojo 2 application.
+---
+
+{% section 'first' %}
+
+# Using Dojo 1 Dijits
+
+## Overview
+
+This tutorial will extend the the [Form widgets](../005_form_widgets/) tutorial.  In this tutorial, we will be taking some of the form widgets and replacing them with Dojo 1 Dijits.
+
+While we would hope that developers would author their code fully in Dojo 2, we recognize that it might be challenging to do a wholesale migration, and that using custom Dijits that have been developed for Dojo 1 might help make it _easier_ to migrate an application, migrating complex Dojo 1 Dijits over a period of time.
```

Comment:
Maybe this instead: To achieve maximum performance and efficiency, developers should eventually author their Dojo 2 application with Dojo 2 widgets. This approach is provided as a migration strategy, to leverage custom Dojo 1 Dijits within a Dojo 2 application, to help make it _easier_ to migrate an application in phases over time.